frontend/src/metabase/admin/datamodel/components/PartialQueryBuilder.jsx

```diff
@@ -124,6 +124,7 @@ export default class PartialQueryBuilder extends Component {
             <a
               data-metabase-event={"Data Model;Preview Click"}
               target={window.OSX ? null : "_blank"}
+              rel="noopener noreferrer"
               className={cx("Button Button--primary")}
               href={previewUrl}
             >{t`Preview`}</a>
```

Comment:
This didn't seem like an external link, so I simply added `rel` directly here to stop linter from complaining. Please let me know if I should've done differently.